release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## August 15, 2019
+
+The following updates were made in this release:
+
+**Android Core 1.4.3**
+
+* Fixed a bug where the database operation may fail on Android Q.
```

Comment:
- Please change "may" to "might".
- Please reword the 2nd bullet so that it reads as follows: 
"Fixed a crash that was happening on Android 8/8.1 and was related to Android's `TimeZoneNamesImpl`."